pkg/nodeoperations/controller.go

```diff
@@ -56,6 +56,14 @@ var (
 	}
 )
 
+func getMTaint() corev1.Taint {
+	return mTaint
```

Comment:
`mTaint` is not a global var. Do we really need it?

Reply:
Should I remove `rTaint` too?

Reply:
Yes